tests/tests/test-precompile/test-precompile-wormhole.ts

```diff
@@ -209,6 +210,21 @@ describeDevMoonbeam(`Test local Wormhole`, (context) => {
       .signAndSend(alith);
     await context.createBlock();
 
+    // we also need to disable the killswitch by setting the 'enabled' flag to Some(true)
+    const ENABLED_FLAG_STORAGE_ADDRESS =
+      "0xb7f047395bba5df0367b45771c00de502551bba17abb82ef3498bab688e470b8";
```

Comment:
Sorry to be tedious but can you derive this instead, so that it's clear how this is derived?

This is to help future generations having to work out where that key is from (and it's a pain to work backwards from a hash).

```
  const enabledFlagKey = u8aToHex(
            u8aConcat(xxhashAsU8a("gmp", 128), xxhashAsU8a("PrecompileEnabled", 128))
          );
```

> :information_source: both functions are available in the `@polkadot/util` repo

Reply:
Good suggestion. Added in 751af83. I also left the keys themselves for future generations :grin: